index.html

```diff
@@ -225,6 +225,57 @@ <h4>Install the client on your headset</h4>
+                                    <img src="screenshots/settings/foveation.jpg" class="d-block w-100">
+                                </div>
+                                <div class="tab-pane fade" id="bitrate" role="tabpanel" aria-labelledby="bitrate-tab">
+                                    <p>Bitrate of the video, in bit/s. Split among decoders based on size and codecs.</p>
+                                    <p>A higher bitrate directly improves image quality by reducing pixelation in fast-paced scenes or in complex visuals, such as landscapes with dense foliage. However, it may also increase latency or overload the network, encoder, or decoder.</p>
+                                    <p>For testing purposes, a bitrate of 50 Mbps is a good starting point for wireless connections. Wired connections, if supported by the hardware, should be able to handle up to 500 Mbps.</p>
+                                    <img src="screenshots/settings/bitrate.jpg" class="d-block w-100">
+                                </div>
+                                <div class="tab-pane fade" id="encoders" role="tabpanel" aria-labelledby="encoders">
+                                    <p>Encoders compresses video data in real-time to reduce bandwidth and latency, enabling smooth transmission to the headset.</p>
+                                    <p>Default value: 3 encoders in order 1/8th, 3/8th and 4/8th of the image.</p>
+                                    <p>WiVRn has the ability to split the video in blocks that are processed independently, this may use resources more effectively and reduce latency. All the provided encoders are put into groups, groups are executed concurrently and items within a group are processed sequentially.</p>
+                                    <p>It is possible to mix CPU/GPU encoding if you have spare CPU cores. On AMD systems, this can provide a significant improvement. For high bitrates, avoid software encoding, as CPU time increases significantly.</p>
+                                    <h5><strong>Encoder</strong></h5>
+                                    <p>Default value: <mark>nvenc</mark> if Nvidia GPU and compiled with nvenc, <mark>vaapi</mark> for all other GPU when compiled with ffmpeg, else <mark>x264</mark>.</p>
+                                    <p>Identifier of the encoder, one of</p>
+                                    <ul>
+                                        <li><mark>x264:</mark> software encoding</li>
+                                        <li><mark>NVENC:</mark>  Nvidia hardware encoding</li>
+                                        <li><mark>VAAPI:</mark>  AMD/Intel hardware encoding</li>
+                                        <li><mark>Vulkan:</mark>  experimental, for any GPU that supports Vulkan video encode</li>
+                                    </ul>
```

Comment:
The <mark> is used in configuration.md to express that a value is used verbatim in the JSON.

In the page here it looks odd, we can just have plain text.